### DISCUSSION TITLE
I want hrefs in all elements

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#155] [Wants]
During the development of XHTML2, there was a proposal to allow any element to have an `href` (rather than limiting it to only `a` and `link`). I’d like to see us revisit that so we can add links declaratively to other elements without involving JavaScript.

<hr>

https://webwewant.fyi/wants/19/

**Comment by mincerafter42** [comment]
Is there a reason you wouldn't be able to accomplish this by placing your desired element inside an `<a>`?

**Reply by aarongustafson** [reply]
You can technically wrap `a` around block elements now (which wasn’t valid in the before times), but some folks would still like to make anything linkable.

**Reply by meyerweb** [reply]
> You can technically wrap `a` around block elements now (which wasn’t valid in the before times), but some folks would still like to make anything linkable.

It’s me, I’m some people.
